docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Dockerfile
+#
+# @author zhibiao.pan@bitmain.com
+# @since 2016-01
+#
+#
+FROM phusion/baseimage:0.9.16
```

Comment:
Without debating the relative merrits of phusion's base images, I can say for sure the relay network stuff should work fine on the regular base images, and in the interest of reducing trust and resource requirements, I'd prefer if you used the docker-provided images.